controllers/user.js

```diff
@@ -830,11 +833,11 @@ async function getRemovalEnrollEndedPage(req, res) {
 async function getRemovalPage(req, res) {
   const user = req.user;
 
-  if (!checkIfEnrolledInRemovalPilot(user) && !req.query.show) {
+  if (!checkIfEnrolledInRemovalPilot(user) && !FormUtils.canShowViaParams(req.query?.show)) {
     return res.redirect("/user/remove-enroll");
   }
 
-  if (checkIfRemovalPilotEnded(user) && !req.query.show) {
+  if (checkIfRemovalPilotEnded(user) && !FormUtils.canShowViaParams(req.query?.show)) {
```

Comment:
Looks like we're checking `!FormUtils.canShowViaParams(req.query?.show)` 2&times; here.

Not sure if we should do something like:

```js
if (!FormUtils.canShowViaParams(req.query?.show)) {
  if (!checkIfEnrolledInRemovalPilot(user)) {
    return res.redirect("/user/remove-enroll");
  }
  if (checkIfRemovalPilotEnded(user)) {
    // if the pilot is over, redirect to the end screen
    return res.redirect("/user/remove-pilot-ended");
  }
}
```

Reply:
I'm up for that.